### ISSUE TITLE
Hadrian has been merged into GHC proper

### ISSUE THREAD

**Issue by alpmestan** [snowleopard/shaking-up-ghc#705]
Hello everyone,

hadrian has been merged into GHC proper, just today!  You can see the commit here: https://github.com/ghc/ghc/commit/94756201349685a34c4495addd3484fdfcc8b498

From now on, let's:
- submit new issues on trac: https://ghc.haskell.org/trac/ghc/newticket ; the hadrian ones should probably have a "hadrian" tag in them, to make it easy to search for all the hadrian-related issues etc.
- submit new patches on phabricator: https://ghc.haskell.org/trac/ghc/wiki/Phabricator; our hadrian patches will just be like any other GHC patch.

We can perhaps keep the active discussions here until they're addressed, and think about moving some important but older & "colder" issues to trac?

**Comment by snowleopard** [comment]
That's great! 

Yes, let's continue active on-going discussions here, but all others should now go to Trac.

@alpmestan I suggest that we prefix all Hadrian-only commit messages with `Hadrian: `, so it's easy to differentiate them from others.

**Comment by alpmestan** [comment]
Yes, that would definitely be a good convention. I already see it done for rts and other things, so it'll fit just nicely.

**Comment by alpmestan** [comment]
Yeah, sorry about that, Ben just couldn't find any way to prevent those.
